README.md

```diff
@@ -0,0 +1,219 @@
+- About two years in, I got freaked out about being pigeon-holed as a programmer in a cubicle for the rest of my life. In a moment of panic, I switched my major to Electrical Engineering (EE).
+- Focused my EE degree on antenna design, lasers, and optics.
+- Loved being in college so much that I stuck around for half of a super-duper senior year. I think anyone who only goes for a fifth year of undergrad isn't truly dedicated to the cause.
+
+### Chapter 3: Various
+
+- Moved back to NOVA and took a job as an EE with a large government contractor.
+- Spent the next three years working on full-scale military flight simulators and sliding into misery.
+- Met my now-wife and let her convince me we should both quit our jobs we didn't like. She took a traveling contract gig and we set off. I had zero plan about what I would do for my career. The only certainty was that I didn't want to be an EE any longer.
+- Started our fantastic voyage in the booming metropolis of South Boston, VA. I spent those three months contemplating life and future career possibilities.
+- Ended up in Houston, TX for a year. Tried to jumpstart a freelance writing career. Made about $250 total but had some cool experiences.
+- Started bartending and teaching SAT prep classes to help with bills. Realized neither of these were great longterm options and that I should be doing something more with my life.
+- Decided to try to get back into software engineering, my first passion, and move to the mecca for it, San Francisco. Taught myself SQL in a weekend and started coding again in Java.
+
+### Chapter 4: San Francisco
+
+- Joined AngelPoints, a 14-person startup building a platform for Corporate Social Responsibility, as a Professional Services Engineer with the plan of transitioning to a Software Engineer role. Fell in love with mission-driven business, building modern software, and San Francisco.
+- AngelPoints is acquired, VP of Eng leaves, sadness ensues.
+- Joined Opower, a rapidly growing mid-sized startup focused on driving energy efficiency using behavioral science, as an Associate Professional Services Engineer.
+- Spent the next 2.5 years rising through the PSE ranks and taking on some of the most complex projects the group faced. Began managing a delivery team with several direct reports.
+- Transitioned into Engineering to manage a team focused on internal tools development. Spent about six months as an individual contributor and learning the tech stack before fully transitioning into EM role. Spent my last two years at Opower managing this team in various incarnations and working on a handful of applications
+- Oracle acquires Opower, a huge chunk of Engineering leaves including my manager, sadness ensues.
+- Joined Optimizely, a rapidly growing mid-sized startup focused on digital experimentation, as an Engineering Manager for the Frontend Engineering team.
+- ?
+
```

Comment:
In later drafts of this readme, i'd be interested to know more about, how working at Optimizely has influenced your management philosophy

Reply:
+1!

Reply:
Interesting idea. Will have to think about how to manage this. I am not sure my philosophy has shifted much since I joined Optimizely. In fact, most of what's in that section above was written elsewhere before I joined. I'm just now getting around to putting it in a public place.